cibuildwheel/linux.py

```diff
@@ -80,10 +87,11 @@ def build(project_dir, output_dir, test_command, test_requires, before_build, bu
                 if [[ "$built_wheel" == *none-any.whl ]]; then
                     # pure python wheel - just copy
                     mv "$built_wheel" /tmp/delocated_wheel
+                    delocated_wheel=(/tmp/delocated_wheel/*.whl)
                 else
-                    auditwheel repair "$built_wheel" -w /tmp/delocated_wheel
+                    auditwheel repair --plat {platform_tag} "$built_wheel" -w /tmp/delocated_wheel
```

Comment:
@mayeut I saw your PR pypa/manylinux#293 adding `AUDITWHEEL_PLAT` was merged. Do you suggest removing the `--plat` flag (being simpler), or do you advise keeping it here?